docs/user-guide/work-packages/duplicate-move-delete/README.md

```diff
@@ -1,16 +1,22 @@
 ---
 sidebar_navigation:
-  title: Copy, move, delete work packages
+  title: Duplicate, move, delete work packages
   priority: 960
-description: How to copy, change project or delete a work package.
-keywords: copy work package, delete work package, move work package
+description: How to duplicate, move to another project or delete a work package.
+keywords: copy work package,duplicate work package, delete work package, move work package
 ---
 
-# Copy, change project or delete a work package
+# Duplicate, move to another project or delete a work package
 
-If you right-click in a work package table, editing options will be displayed. Here, you can change project, copy the work package, or copy its URL to the clipboard, delete it or move it to another project.
+> [!TIP] 
+>
+> In OpenProject 14.5 the term *Copy a work package* was replaced by *Duplicate a work package*.
+>
+> *Change project* was replaced by *Move to another project*.
 
-![change copy delete work package](openproject_user_guide_copy_move_delete_wptableview.png)
+If you right-click in a work package table, editing options will be displayed. Here, you can move a work package to another project, duplicate a work package, or copy its URL to the clipboard, delete it or duplicate it to another project.
```

Comment:
Here, I would rephrase the last bit: "... duplicate it in another project" (vs "to").